Purchases/SwiftSources/Purchasing/PurchaserInfoHelper.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  PurchaserInfoHelper.swift
+//  Purchases
+//
+//  Created by RevenueCat.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc public class PurchaserInfoHelper: NSObject {
```

Comment:
since the class doesn't directly use purchaserInfo, maybe `TransactionsFactory` would be more accurate?

Reply:
Sounds good! I thought about using it to build other stuff in the PurchaserInfo, but we can think about that in the future